cloudbreak/stack/stack_template.go

```diff
@@ -126,9 +126,10 @@ func generateStackTemplateImpl(mode cloud.NetworkMode, stringFinder func(string)
 	template := model.StackV2Request{
 		Cluster: &model.ClusterV2Request{
 			Ambari: &model.AmbariV2Request{
-				BlueprintName: "____",
-				UserName:      &(&types.S{S: "____"}).S,
-				Password:      &(&types.S{S: ""}).S,
+				BlueprintName:     "____",
+				UserName:          &(&types.S{S: "____"}).S,
+				Password:          &(&types.S{S: ""}).S,
+				ValidateBlueprint: &(&types.B{B: false}).B,
```

Comment:
This behavior overrides a backend behavior without the user expicitly asked the cli to override it. Business logic is implemented here, because backend's responsible to select validation model.

Reply:
Changing the backend api behavior could be a lot more risky. It would change the behavior of the existing templates.